### PR TITLE
LIME-369: Goal banner is displayed without content on Goals page when screen width is 767 and less and if no goals are in progress

### LINKED ISSUE #369: BUG: Goal banner is displayed without content on Goals page when screen width is 767 and less and if no goals are in progress
### Describe your issue

Banner is displayed with no content when screen width is 767 and less if no goals are in progress and no progress bar may be displayed 

### Steps to reproduce

Pre-condition: user is logged in on https://live-in-motion-everyday.com/, responsive mode with screen width 758 is selected in Toggle device in Devtools
1. Navigate to Goals page
2. Minimize screen width to 757
3. Pay attention if banner is displayed with content

### What was the actual result?

Banner is displayed with no content when screen width is 757 and less if no goals are in progress and no progress bar may be displayed 

### What was the expected result?

Banner is displayed with content regardless the screensize. If no goals are in progress and no progress bar may be displayed, the only label Unleash your fitness potential with new goals! is displayed

### Put here any screenshots or videos (optional)

image

### What OS are you seeing the problem on?

Windows

### What browsers are you seeing the problem on?

Chrome

### Severity

Minor (S4)

### PR DESCRIPTION
**Pull request by SergiiTernytskyi**
https://www.loom.com/share/fef16d3571eb493c92f4a2f74d0353f7

### CODE REVIEW

**Comment by nikita-remeslov** [comment]
Content should be centered, check Overview page

**Comment by SergiiTernytskyi** [comment]
> Content should be centered see Overview page

I know, but It is ticket https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/issues/372, so I did not fixed it.

**Comment by SergiiTernytskyi** [comment]
> Content should be centered, check Overview page

For this purpose we have https://github.com/BinaryStudioAcademy/bsa-winter-2023-2024-lime/issues/372. Or should I change here?

File: frontend/src/bundles/overview/components/goal-widget/goal-widget.tsx [hunk 1]
```diff
@@ -40,7 +40,9 @@ const GoalWidget = ({
             )}
         >
             <div>
-                <p className="font-heavybold text-lm-black-200 hidden text-[1.5rem] leading-7 md:block">
+                <p
+                    className={`font-heavybold text-lm-black-200 text-xl leading-7 md:text-[1.5rem]  ${hasAchievement ? 'hidden md:block' : ''}`}
```
Comment (dimapopovych) [inline frontend/src/bundles/overview/components/goal-widget/goal-widget.tsx]: let's use getValidClassNames helper for combining styles

**Review by dimapopovych — CHANGES_REQUESTED** [review]

**Review by valery-chumak — APPROVED** [review]

**Review by jhladkov — APPROVED** [review]

**Review by ArtemLymarenko — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**